lib/rules/match-exported.js

```diff
@@ -34,16 +56,18 @@ module.exports = function(context) {
                 exportedName = getExportedName(node),
                 isExporting = Boolean(exportedName),
                 expectedExport = getStringToCheckAgainstExport(parsed),
+                transformed = transform(exportedName, context),
+                transformedText = transformed ? ' (transformed)' : '',
```

Comment:
This can probably always be `(transformed)` since the user clearly sees both values and can determine whether they are transformed or not.